docs/rhel-custom-image.md

```diff
@@ -0,0 +1,57 @@
+# RedHat Enterprise Linux
+
+Cloud providers which are listed below, support using rhel as an operating system option: 
+
+- AWS 
+- Azure
+- GCE
+- KubeVirt
+- Openstack
+- vSphere
+
+####  AWS:
+For amazon web service cloud provider, one the user applies for Gold Accesses Images using their RedHat Customer Portal, their request 
+will be reviewed and validated. Afterwards, new images will be added to the aws account under EC2-> Images-> AMIs-> Private Images.
+Once the images are available in the aws account, the image id for rhel(at the moment we only support 8.x.x versions) should be then 
```

Comment:
Instead of specifying the supported version I would add a link to the supported OS versions, to avoid double mainteanance.

Reply:
Do you mean to this file here: https://github.com/kubermatic/machine-controller/blob/master/docs/operating-system.md

Reply:
yes